src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -941,7 +975,37 @@ private PhysicalLocation CreatePhysicalLocation(string uri, Region region = null
                 Region = region,
                 ContextRegion = region
             };
+        }
+
+        private static readonly Regex LogicalLocationRegex =
+            new Regex(
+                @"
+                  ([^\s]*\s+)?         # Skip over an optional leading blank-terminated return type name such as 'void '
+                  (?<fqln>[^(]+)       # Take everything up to the opening parenthesis.
+                ",
+                RegexOptions.Compiled | RegexOptions.ExplicitCapture | RegexOptions.IgnorePatternWhitespace);
```

Comment:
>IgnorePatternWhitespace [](start = 84, length = 23)

oh! is this setting what enables that feature?

Reply:
Yes!

---
In reply to: [282150462](https://github.com/microsoft/sarif-sdk/pull/1480#discussion_r282150462) [](ancestors = 282150462)